int64 highScore
int64 roundsComplete
int64 losses